src/main/resources/application-local.yml

```
@@ -78,13 +78,13 @@ runner:
 docker:
   build:
     dir: /code    # Working directory to construct in docker image
-    timeout: 10    # Timeout, in seconds, of docker build process
+    timeout: 300    # Timeout, in seconds, of docker build process

```

This needs to be reset back to something lower :).

---

Really depends on how fast the system creates the image from scratch. Unless we have premade images, then the number can go back down to 10. Perhaps add a note to change this before prod and to figure a solution to this issue.
```suggestion
    # TODO: Fix long image creation time for images being created for the first time.
    timeout: 300    # Timeout, in seconds, of docker build process

```

---

Yeah, you know what for locally with people running it for the first time it's not a big deal we can leave it at 300. The prod configuration will be managed externally anyways. We should include a base Haskell file though.